### PR TITLE
Library

### PR DESCRIPTION
**Pull request by oluwoleilesanmi**
Kindly help review

### CODE REVIEW

File: assests/js/main.js [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+      return localStorage.setItem(namespace, JSON.stringify(data));
+    } else {
+      let store = localStorage.getItem(namespace);
+      return (store && JSON.parse(store)) || [];
+    }
+  },
+  
+  validateInputs: function(author, title, pages) {
+   const message = "Can't be Empty";
+   (pages =='') ?
```
Comment (heyapricot) [inline assests/js/main.js]: Why `==` and not `===` ?

**Comment by oluwoleilesanmi** [comment]
- Hi Alvaro all issues have been corrected. 
- I have also taken note of the createElement best practice, i will implemented this going forward in all projects i havnt used innerHtml, and later in this one.
- An array is also used for storage as declared in line 46 instantiated in line 72.
- Thanks.

File: assests/js/main.js [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+  this.author = author;
+  this.title = title;
+  this.pages = pages;
+}
+Book.prototype.getAuthor = function() { return this.author;};
+Book.prototype.getStatus = function() { return this.status;};
+Book.prototype.getTitle = function() { return this.title;};
+Book.prototype.getPage = function() { return this.pages;};
+
+let util = {
```
[the remark this answers is not in the document]
Comment (oluwoleilesanmi) [reply assests/js/main.js]: closed

File: assests/js/main.js [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+  store: function() {
+    return [];
+  },
+
+  validateInputs: function(author, title, pages) {
+    const message = "Can't be Empty";
+    pages == ""  ? util.validationdisplay(message, "pages") : this.validationdisplay("", "pages");
```
[the remark this answers is not in the document]
Comment (oluwoleilesanmi) [reply assests/js/main.js]: closed

**Comment by oluwoleilesanmi** [comment]
@rvvergara @heyapricot  thanks for these swift reviews and i have taken note of your points, to argue constructively on this matter

- That i have not implemented a correction of a snag doesn't mean you have not impacted Knowledge.
- I believe these things which i have highlighted earlier on our group channel is a difference of ideas in how we lead as people and how we want to be lead and i respect your approach.
- I have implemented some of what you asked to show that i respect your opinion even while i think those changes should not hold this project back.
- In the end when we start working its going to be about results, the question is always does it work no matter how rough for a start.
- Projects on the job also have deadlines so time is always a factor. In production we also have to trade refactoring for time.
- I believe thumbs up is a fiction of our imagination, the knowledge gained is really for us to keep. but i encourage you keep the rough parts of my code coming.

:star: Except you will be able argue against the points stated above beyond reasonable doubt i stand by my opinion and approach. Thanks

**Pull request merged**